### PR TITLE
Control: added pad_msg to local_view and updated control sub modules

### PR DESCRIPTION
**Pull request by sjiang2018**

### CODE REVIEW

File: modules/control/dag/lateral_longitudinal_module.dag [hunk 1]
```diff
@@ -43,15 +41,18 @@ module_config {
...
                 {
-                    channel:"/apollo/control"
+                    channel:"/apollo/control/precommand"
                 }
             ]
         }
     }
-}
+
+
```
Comment (xiaoxq) [inline modules/control/dag/lateral_longitudinal_module.dag]: Remove extra blank lines.

File: modules/control/dag/lateral_longitudinal_module.dag [hunk 2]
```diff
@@ -43,15 +41,18 @@ module_config {
...
+}
```
Comment (xiaoxq) [inline modules/control/dag/lateral_longitudinal_module.dag]: Add `\n`

File: modules/control/submodules/mpc_controller_submodule.cc [hunk 3]
```diff
@@ -80,18 +84,25 @@ bool MPCControllerSubmodule::Proc(
 
 Status MPCControllerSubmodule::ProduceControlCommand(
     ControlCommand* control_command) {
-  std::lock_guard<std::mutex> lock(mutex_);
+  //   std::lock_guard<std::mutex> lock(mutex_);
```
Comment (xiaoxq) [inline modules/control/submodules/mpc_controller_submodule.cc]: Remove if not needed, or comment why we don't need it temporally.

File: modules/control/submodules/preprocessor_submodule.cc [hunk 4]
```diff
@@ -64,55 +64,49 @@ bool PreprocessorSubmodule::Init() {
...
   Status status = CheckInput(local_view_);
 
   if (!status.ok()) {
     AERROR_EVERY(100) << "Control input data failed: "
                       << status.error_message();
-    preprocessor_status->mutable_engage_advice()->set_advice(
+    control_preprocessor->mutable_engage_advice()->set_advice(
         apollo::common::EngageAdvice::DISALLOW_ENGAGE);
-    preprocessor_status->mutable_engage_advice()->set_reason(
+    control_preprocessor->mutable_engage_advice()->set_reason(
```
Comment (xiaoxq) [inline modules/control/submodules/preprocessor_submodule.cc]: Save `mutable_engage_advice` to a temp var and reuse.

File: modules/control/submodules/mpc_controller_submodule.cc [hunk 5]
```diff
@@ -57,12 +57,16 @@ bool MPCControllerSubmodule::Init() {
   }
   mpc_controller_conf_.mutable_mpc_controller_conf()
       ->set_allocated_calibration_table(&calibration_table_);
+
+  control_command_writer_ =
```
Comment (Capri2014) [inline modules/control/submodules/mpc_controller_submodule.cc]: Why control command writer here?

File: modules/control/submodules/lat_lon_controller_submodule.cc [hunk 6]
```diff
@@ -60,6 +60,10 @@ bool LatLonControllerSubmodule::Init() {
         "Control init longitudinal controller failed! Stopping...");
     return false;
   }
+
+  control_command_writer_ =
+      node_->CreateWriter<ControlCommand>(FLAGS_control_pre_command_topic);
+
```
Comment (Capri2014) [inline modules/control/submodules/lat_lon_controller_submodule.cc]: Why create control command writer with `FLAGS_control_pre_command_topic` ?

File: modules/common/adapters/adapter_gflags.cc [hunk 7]
```diff
@@ -40,6 +40,8 @@ DEFINE_string(control_preprocessor_topic, "/apollo/control/preprocessor",
               "control preprocessor topic name");
 DEFINE_string(control_local_view_topic, "/apollo/control/localview",
               "control local view topic name");
+DEFINE_string(control_pre_command_topic, "/apollo/control/precommand",
```
Comment (Capri2014) [inline modules/common/adapters/adapter_gflags.cc]: How it is defined? Try match the proto definition with topic definition to avoid confusion.

**Comment by Capri2014** [comment]
@sjiang2018 @xiaoxq Will merge this one in as hotfix and do a through refine.

**Pull request merged**